core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -1,11 +1,7 @@
 {% extends "base.html" %}
 
 {% block maintitle %}
-  {% if title %}
-    {{ title }} - Oppia Editor
-  {% else %}
-    Untitled Exploration - Oppia Editor
-  {% endif %}
+  Create an Exploration -- Oppia
```

Comment:
Actually, let's leave this one as it was before. The previous version seems more descriptive.